tests/Behat/features/add-block-element-to-data-object.feature

```diff
@@ -0,0 +1,33 @@
+@javascript
+Feature: Add elements in the CMS DataObject
+  As a CMS user
+  I want to add elements in the CMS DataObject
+  So that I can use multiple elements in a Dataobject
+  
+  Background:
+    Given I am logged in with "ADMIN" permissions
+
+@runthis
+  Scenario: I can add inline-editable elements to the DataObject
+    When I go to "/admin/test-blog-posts-admin/"
+    Then I press the "Add Test Blog Post" button
+    Then I press the "Create" button
+    Then I wait 1 second
+    Then I press the "Add block" button
+    Then I press the "Content" button in the add block popover
+    Then I should see "Untitled Content block" as the title for block 1
+    When I click on block 1
+    And I should see "Title"
+
+@runthis2
+  Scenario: I can add non-inline-editable elements to the DataObject
+    When I go to "/admin/test-blog-posts-admin/"
+    Then I press the "Add Test Blog Post" button
+    Then I press the "Create" button
+    Then I wait 1 second
+    Then I press the "Add block" button
+    Then I press the "Form" button in the add block popover
```

Comment:
Where does the "form" button come from? If that's the userform elemental block, that's not included in this module's dev dependencies.